applications/FluidDynamicsApplication/python_scripts/check_and_prepare_model_process_fluid.py

```diff
@@ -32,6 +32,15 @@ def __init__(self, main_model_part, Parameters ):
         #self.list_of_inlets = Parameters["list_of_inlets"]
 
 
+    def _ElementsAreNotSimplex(self):
+        "Checks whether the first element is non-simplex"
+        if self.main_model_part.NumberOfElements() == 0:
+            return False
```

Comment:
```suggestion
            return True
```

otherwise again the check is skipped if there are no elements